doc/nrf/ug_gzll.rst

```diff
@@ -38,6 +32,7 @@ Gazell provides the following features:
 
 * Capability to control the RF front-end module.
 * Generates transmission statistics for each RF channel.
+* Several different networks (see `Gazell network`_).
```

Comment:
Do not think supporting several networks is intended in the design of nRF24L01. The two ESB base addresses are for 'pairing' and 'normal' data transfers. It is a star network.

How about removing this clause? Keep it low profile.

Reply:
Okay, I agree. If it is something that is not intended then it should not be mentioned. 

Does this mean that the entire section should be removed? Or just remove this line. @leewkb4567

Reply:
Just the line on several networks. Not the entire section. Thanks.

Reply:
Okay, will do. Thank you!